hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/keygen/CustomKeyGenerator.java

```diff
@@ -83,7 +83,7 @@ private static List<BuiltinKeyGenerator> getPartitionKeyGenerators(List<String>
       return Collections.emptyList();
     } else {
       return partitionPathFields.stream().map(field -> {
-        String[] fieldWithType = field.split(CustomAvroKeyGenerator.SPLIT_REGEX);
+        String[] fieldWithType = field.split(BaseKeyGenerator.CUSTOM_KEY_GENERATOR_SPLIT_REGEX);
```

Comment:
nit: just `CUSTOM_KEY_GENERATOR_SPLIT_REGEX` should suffice

Reply:
Addressed